NINJ2.py

```diff
@@ -0,0 +1,645 @@
+import os
+from Bio import SeqIO
+from Bio.SeqUtils import GC
+
+
+def fastq_filtrator(input_path: str, output_filename=None,
```

Comment:
Просто рассуждение: мне видится удобнее, когда имя выходного файла после остальных аргументов, не придётся его каждый раз указывать при смене параметров фильтрации
```suggestion
def fastq_filtrator(input_path: str, 
                    output_filename: str | None = None,
```